tests/recipes/rational_replicate.rs

```diff
@@ -1,36 +1,51 @@
 use super::*;
-use ruler::enumo::{Filter, Ruleset, Workload};
+use ruler::{
+    enumo::{Filter, Ruleset, Workload},
+    recipe_utils::{base_lang, iter_metric, run_workload},
+};
 
 pub fn replicate_ruler1_recipe() -> Ruleset<Math> {
     let mut rules = Ruleset::default();
     let limits = Limits::default();
 
     // Domain
-    let lang = Workload::new(&["var", "const", "(uop expr)", "(bop expr expr)"]);
     let vars = &Workload::new(["a", "b", "c"]);
     let consts = &Workload::new(["0", "-1", "1"]);
     let uops = &Workload::new(["~", "fabs"]);
     let bops = &Workload::new(["+", "-", "*", "/"]);
+    let lang = base_lang()
```

Comment:
Any reason we're using Atoms-based enumeration on the other 3 recipes, but still doing the layering thing here? Doesn't matter to me as long as the results match up, just wondering.

Reply:
no real reason- I vaguely remember atoms not as well last fall, but i didn't try it tbh